scipy/stats/stats.py

```diff
@@ -3692,6 +3695,136 @@ def f_oneway(*args, axis=0):
     return F_onewayResult(f, prob)
 
 
+def alexandergovern(*args):
+    """
+    Performs the Alexander Govern test.
+
+    The Alexander-Govern approximation tests the equality of k independent
+    means in the face of heterogeneity of variance. The test is applied to
+    samples from two or more groups, possibly with differing sizes.
+
+    Parameters
+    ----------
+    sample1, sample2, ... : array_like
+        The sample measurements for each group.  There must be at least
+        two samples.
+
+    Returns
+    -------
+    statistic : float
+        The computed A statistic of the test.
+    pvalue : float
+        The associated p-value from the chi-squared distribution.
+
+    Notes
+    -----
+    The use of this test relies on several assumptions.
+
+    1. The samples are independent.
+    2. Each sample is from a normally distributed population.
+    3. Unlike `f_oneway`, this test does not assume on homoscedasticity,
+       instead relaxing the assumption of equal variances.
+    4. The input samples are one dimentional with size greater than one.
```

Comment:
```suggestion
    4. The input samples are one dimensional with size greater than one.
```
I'm confused... how did this get un-fixed?